src/platform/i386/kernel.c

```diff
@@ -67,14 +69,19 @@ kern_memory_setup(struct multiboot *mb, u32_t mboot_magic)
 	for (i = 0 ; i < mb->mods_count ; i++) {
 		struct multiboot_mod_list *mod = &mods[i];
 
```

Comment:
Is there a blank line after this definition?  We don't want blank lines between different defintions.